builders/manage/dapp/manage-tokens.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Manage Tokens
+description: Learn how the appchain governor can manage the token of a Tanssi-powered appchain using the dApp to mint them, configure the EIP-1559 fee market, and more.
+---
+
+# Manage Tokens
+
+## Introduction {: #introduction }
+
+Any appchain deployed through Tanssi is sovereign and free to define the governance model that best fits its use case. The appchain governor has superpowers over the chain administration, meaning privileged functions, such as updating the runtime, fall within his reach.
+
+There are some actions related to the native token management that are available to the appchain governor on the Tanssi dApp:
+
+- **Mint Tokens** - mints new tokens, increasing the total supply
+- **Update Balances** - increases or decreases the balance of an account, affecting the total supply
+- **Transfer Tokens** - executes a forced token transfer from one account to another
+- **Configure Gas Dynamics** - only available on EVM-compatible appchains, this action changes the [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md){target=\_blank} configuration, affecting the transaction pricing mechanism
```

Comment:
I just realized that sometimes we do "Mint Tokens" (both caps) other times we do "Mint tokens" (first word caps) - so we need to start making sure we follow a standard here. Let's just do first word caps